UserOlena/Learnify
# Typedefs resolvers

Set up file structure for typedefs and resolvers where each collection has its own separate file

To test:
- `npm run seed` in the project root folder
- `npm start`
- Open `http://localhost:3001/graphql` in the browser and confirm that each of the queries returns complete data
